### PR TITLE
Add stale .github workflow

### PR DESCRIPTION
**Pull request by w7shdev**
# Issue Addressed 
Discussed in #84 

The bot is added but the implementation is missing 

## Description:

More about stale bot https://github.com/probot/stale

## Testing:

N/A

## Checklist

- [x] I have commented the newly added code
- [ ] I update the Readme.md file to include the new changes to the project
- [ ] All the previously included Unit Tests passed in all the cases

### CODE REVIEW

**Comment by w7shdev** [comment]
@Ma7eer I don't have a permission. so please when you have a time review this PR

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+daysUntilStale: 3
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 1
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - bug
+  - Bug(Potintail)
```
Comment (w7shdev) [inline .github/stale.yml:12]: There is a typo here I can't edit the label since I don't have permissions ;)

Comment (Ma7eer) [reply .github/stale.yml:12]: @w7shdev permission given 😁

**Comment by Ma7eer** [comment]
@w7shdev once you fix the typo mentioned I will do one more review

File: .github/stale.yml (hunk 1 above)
Comment (w7shdev) [inline .github/stale.yml:12]:
```suggestion
  - Bug(Potential)
```

Comment (w7shdev) [reply .github/stale.yml]: Thanks & done

**Review by Ma7eer — APPROVED** [review]
LGTM

**Pull request merged**